book/pipelines.md

```diff
@@ -168,12 +167,11 @@ See: [Custom Commands -> Pipeline Input](custom_commands.html#pipeline-input)
   Example:
 
   ```nushell
-  > 4               # Pipeline input
+    4               # Pipeline input
```

Comment:
Looking at the full doc, this section is kind of wonky in and of itself. The list of rules starts un-indented with rule 1, with the example indented two spaces. Rule 1.5 onwards has a two-space indent that I don't remember adding, and rule 2 starts with a `*` instead of a `-` that I definitely didn't add.

I'll back through this section manually and give them a uniform layout.

Reply:
Thanks - 1.5 and 2.5 were sub-indented under 1 and 2.  So the code-blocks were indented differently as well.  I'll double-check it once we merge.  I'm guessing the `*` was a change that the formatter made behind your back when you pushed ;-).  It messes with me quite often as well.

Reply:
Yeah - In looking at it now that it's published, I think your simplification just works better.